iOSTest/View Controllers/LoginViewController.swift

```diff
@@ -82,7 +82,7 @@ class LoginViewController: UIViewController {
         
         // Both username and password are valid - go to MenuViewController
         else {
-            let validAlert = UIAlertController(title: "Login Successful", message: "Function duration: \(runtime)", preferredStyle: .alert)
+            let validAlert = UIAlertController(title: "Login Successful", message: "Function duration: \(runtime) ms", preferredStyle: .alert)
```

Comment:
I've encountered a bug. Please review the simulator screen grab.

![Simulator Screen Shot - iPhone 8 - 2020-06-13 at 00 34 33](https://user-images.githubusercontent.com/16193288/84560019-d81a3e00-ad0d-11ea-9347-1415c07dfcb5.png)